frontend/src/features/permits/apiManager/permitsAPI.ts

```diff
@@ -382,11 +382,12 @@ export const getCurrentAmendmentApplication = async (
 /**
  * Retrieve the list of active or expired permits.
  * @param expired If set to true, expired permits will be retrieved.
+ * @param paginationOptions The pagination and filters applied.
  * @returns A list of permits.
  */
 export const getPermits = async (
   { expired = false } = {},
-  { page = 0, take = 10 }: PaginationOptions,
+  { page = 0, take = 10, searchValue, sorting = [] }: PaginationAndFilters,
```

Comment:
is `maxValue` is `25` should this default `take` value be that?

Reply:
No, the default page size is only 10. The user can increase it to 25 if they choose to.